src/index.ts

```diff
@@ -391,6 +391,15 @@ svg.rs-svg-container {
           },
         },
       },
+      {
+        id: "disable-global-trigger",
+        name: "Disable Global Node Menu Trigger",
+        action: {
+          type: "switch",
+        },
+        description:
+          "Whether to disable the global trigger for the Discourse Node Menu, found at [[roam/js/discourse-graph]], and use the custom trigger instead",
+      },
```

Comment:
Why have a disable setting over just the custom trigger setting (where empty/obscure string effectively disables)?

Reply:
changed to setting